android/src/main/java/com/github/godness84/RNRecyclerViewList/RecyclerViewBackedScrollView.java

```diff
@@ -205,6 +207,42 @@ public void onViewRecycled(ConcreteViewHolder holder) {
             ((RecyclableWrapperViewGroup) holder.itemView).removeAllViews();
         }
 
+        @Override
+        public void onViewAttachedToWindow(@NonNull ConcreteViewHolder holder) {
+            ArrayList<View> allEditTextsInViewGroup = getAllEditTextChildren(holder.itemView);
+            for (View child : allEditTextsInViewGroup) {
+                EditText editText = (EditText) child;
+                // by default, an EditText that is focusable should also be selectable, so
```

Comment:
It seems to me that we're making a strong assumption here. This could work in our case, but I can have an EditText that is focusable but not selectable. 
Should we check `textIsSelectable` before resetting it to true?

Reply:
I agree it is an assumption. Unfortunately there is no query API for this. However, I can't think of a practical  case where it makes sense to use an EditText that is focused but can't be selected. It may make sense for TextView though. We can certainly live without that in this fwiw.